### ISSUE TITLE
ApMon C++17 compatibility

### ISSUE THREAD

**Issue by awegrzyn** [alisw/apmon-cpp#2]
Tested on CC7 and g++ 7.3.1
Fails with multiple errors:
```
../include/ApMon.h:505:27: error: ISO C++1z does not allow dynamic exception specifications
   ApMon(char *initsource) throw(runtime_error);
...
```

**Comment by awegrzyn** [comment]
@costing

**Comment by ktf** [comment]
Looks like the correct solution is also here to simply remove the `throw(runtime_error)` part.

**Comment by dberzano** [comment]
I am taking care of it.

**Comment by dberzano** [comment]
Fixed in tag v2.2.8-alice2 (51b19863a070107dad83bdc0f83c4c0a38257c7e).
